worker/backup_ee.go

```diff
@@ -13,145 +13,68 @@
 package worker
 
 import (
-	"net/http"
-	"time"
+	"context"
 
 	"github.com/dgraph-io/dgraph/ee/backup"
 	"github.com/dgraph-io/dgraph/posting"
 	"github.com/dgraph-io/dgraph/protos/pb"
 	"github.com/dgraph-io/dgraph/x"
 
 	"github.com/golang/glog"
-	"golang.org/x/net/context"
 )
 
-// NOTE: This file is a self-contained source of callers of 'ee/backup' pkg.
+// Backup handles a request coming from another node.
+func (w *grpcWorker) Backup(ctx context.Context, req *pb.BackupRequest) (
```

Comment:
This function is not really doing anything. We can just replace backupCurrentGroup with Backup